### DISCUSSION TITLE
Accidentally downgraded plans after clicking to sign up as a teacher

### DISCUSSION THREAD

**Discussion by grahamzemel** [community/community#38839] [GitHub Education]
Hello, I recently was looking at the different plans and educational options that Github provides and hit the 'sign-up' button on the teacher page, presumably losing my status as a student and therefore my included Github Pro. 

I've had my information submitted 3 times to the Github student developer portal and I'm still currently verified as a student, but I can't access Github Pro at all from the benefits section or it's section in my settings page. 

Any suggestions or help is greatly appreciated, thanks!

**Comment by samus-aran** [comment]
Hi @grahamzemel (again!) 👋 

I can see you still retain your GitHub Pro badge on your profile. Has your issue been rectified or do you require any more assistance? 🙌

**Comment by grahamzemel** [comment]
Yep, I had another team member assist me. Thank you!
